lms/templates/courseware/info.html

```diff
@@ -58,8 +58,9 @@ <h2 class="title">${_("You are not enrolled yet")}</h2>
     <div class="container">
       <div class="home">
         <div class="page-header-main">
-            <h1 class="page-title">${_("Welcome to {org}'s {course_name}!").format(org=course.display_org_with_default, course_name=course.display_number_with_default)}</h1>
-            <h2 class="page-subtitle">${course.display_name_with_default}</h2>
+            <h3 class="page-title">${_("Welcome to {org}'s {course_name}!").format(org=course.display_org_with_default, course_name=course.display_number_with_default)}
+            <div class="page-subtitle">${course.display_name_with_default}</div>
```

Comment:
This seems semantically a little strange, having the subtitle be included inside the ``page-title`` element. What is the benefit of having it structured this way?